### ISSUE TITLE
add themes lol

### ISSUE THREAD

**Issue by capitanbarbosa** [dumbeau/AutoHotPie#155]
be able to import themes so the menu can look like skyrim, or mario kart idk =)

**Comment by dumbeau** [comment]
Absolutely, I know I'm a broken record when I say this, but I'm planning this for V2.  As the appearance settings are better organized so themes can be created more easily.

**Comment by capitanbarbosa** [comment]
Dope dood. Thanks, ur ma hero!! Apps saved me hundreds of hours

Enviado con Gmail Mobile
